saleor/wombat/serializers.py

```diff
@@ -0,0 +1,200 @@
+    cost_price = serializers.SerializerMethodField()
+    quantity = serializers.SerializerMethodField()
+    options = serializers.SerializerMethodField()
+
+    def get_price(self, obj):
+        return obj.get_price().gross
+
+    def get_cost_price(self, obj):
+        # Get maximal cost price for a product
+        stock = sorted(obj.stock.all(), key=lambda s: s.cost_price,
+                       reverse=True)
+        return stock[0].cost_price
+
+    def get_quantity(self, obj):
+        # Get maximal quantity for a product
+        stock = sorted(obj.stock.all(), key=lambda s: s.quantity,
+                       reverse=True)
+        return stock[0].quantity
+
+    def get_options(self, obj):
+        attributes = obj.product.attributes.all()
+        attr_map = {str(a.pk): a.name for a in attributes}
+        options = {}
+        for attribute_id, value in obj.attributes.items():
+            name = attr_map[str(attribute_id)]
+            options[name] = value
+        return options
+
+    class Meta:
+        model = ProductVariant
+        fields = ('sku', 'price', 'cost_price', 'options', 'quantity')
+
+
+class ProductImageSerializer(serializers.ModelSerializer):
+
+    url = serializers.CharField(source='image.url')
+    type = serializers.SerializerMethodField()
+    dimensions = serializers.SerializerMethodField()
+    position = serializers.CharField(source='order')
+    title = serializers.CharField(source='alt')
+
+    def get_type(self, obj):
+        return 'product_image'
+
+    def get_dimensions(self, obj):
+        return {
+            'height': obj.image.height,
+            'width': obj.image.width
+        }
+
+    class Meta:
+        model = ProductImage
+        fields = ('url', 'position', 'title', 'type', 'dimensions')
+
+
+class ProductSerializer(serializers.ModelSerializer):
+    variants = ProductVariantSerializer(many=True)
+    images = ProductImageSerializer(many=True)
+    options = serializers.SerializerMethodField()
+    properties = serializers.SerializerMethodField()
+    shipping_category = serializers.SerializerMethodField()
+    permalink = serializers.URLField(source='get_absolute_url')
+
+    def get_options(self, product):
+        return [a.name for a in product.attributes.all()]
+
+    def get_properties(self, product):
+        return {}
+
+    def get_shipping_category(self, product):
+        return 'standard'
+
+    class Meta:
+        model = Product
+        fields = ('id', 'name', 'description',
+                  'available_on', 'permalink',
+                  # 'meta_description',
+                  # 'meta_keywords',
+                  'shipping_category', 'options',
+                  'properties', 'images', 'variants')
+
+
+class GetWebhookRequestSerializer(serializers.Serializer):
+    request_id = serializers.CharField(required=True)
+    parameters = serializers.DictField(child=serializers.CharField())
+
+
+PAYLOAD_CONFIG = {
+    Order._meta.model: {
```

Comment:
It looks tricky. Why we are using "protected" value `_meta` here? This `PAYLOAD_CONFIG` is used only once in `tasks.py` can we move this logic there?